srcdoc/configure/h2olog.mt

```diff
@@ -0,0 +1,85 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Using h2olog for Tracing")->(sub {
+
+<p>h2olog is a <a href="https://www.kernel.org/doc/html/latest/bpf/index.html">BPF</a> (<a href="https://www.kernel.org/doc/Documentation/networking/filter.txt">kernel doc</a>) backed HTTP request tracing tool for the <a href="https://github.com/h2o/h2o">H2O</a> server.
+h2olog can also be used to log <a href="https://en.wikipedia.org/wiki/QUIC">QUIC</a> events for <a href="https://en.wikipedia.org/wiki/Transport_layer">transport layer</a> observation.
+See <a href="#tracing-quic-events">Tracing QUIC events</a> for how.</p>
```

Comment:
Maybe something like:
h2olog is an experimental BPF**-**backed ~HTTP request~ tracing tool for the H2O server. It can be used for tracing HTTP-level or QUIC-level events.

The reason to state "experimental" is that we do not yet consider the command line interface of h2olog to have been finalized (do we?).

Reply:
refined docs: b46df446bc128c4414d0c430725d43997f485bd6

as well as added h2olog to "Key Features" 7b2c419de4428155dcb33192dbf4faf356c593e4